generators/server/files.js

```diff
@@ -142,7 +142,11 @@ function writeFiles() {
             }
 
             // Thymeleaf templates
-            this.copy(SERVER_MAIN_RES_DIR + 'templates/error.html', SERVER_MAIN_RES_DIR + 'templates/error.html');
+            if (this.angularVersion === 'angular1') {
+                this.copy(SERVER_MAIN_RES_DIR + 'templates/error.html', SERVER_MAIN_RES_DIR + 'templates/error.html');
+            } else {
+                this.copy(SERVER_MAIN_RES_DIR + 'templates/error.component.html', SERVER_MAIN_RES_DIR + 'templates/error.component.html');
```

Comment:
test-creation is to verfiy if the files required are generated for an app and files.js actually decides what are the files that needs to be generated for a app config